doc/irmin-pack/design/lower_layer.md

```diff
@@ -265,40 +265,40 @@ Nothing changes from the current behavior. All remnant files (in both upper and
 
-This scenario is detected by comparing the generation number (the `gen` part of the name) of the volume's control file and the generation in the upper control file. If they are the same then this is resolved by swapping the appendable volume's control file on restart via a file rename. If they are not, it is unlinked.
+This scenario is detected by comparing the generation number (the `gen` part of the name) of the volume's control file and the generation in the upper control file. If they are the same, this is resolved by swapping the appendable volume's control file on restart via a file rename. If they are not, it is unlinked.
 
-This scenario highlights a motivating reason for `volume.gen.control`. It gives us a way to coordinate the control files of the volume and the upper layer and to recover in the scenarios where one or the other fails to update.
+This scenario highlights a motivating reason for `volume.gen.control`. It gives us a way to coordinate both the volume's control files and upper layer and to recover in the scenarios where one or the other fails to update.
 
 **Moment C**
 ![](./imgs/lower_layer/bXuEjOi.png)
 
 
-This moment is after the volume's control file has been renamed but before files have been unlinked (in the upper).
+This moment is after the volume's control file has been renamed but before files have been unlinked (in the upper layer).
 
-Nothing changes from current behavior. Files are unlinked on node restart.
+Nothing changes from current behaviour. Files are unlinked on node restart.
 
 
 ### Validations
 
 The following validations are performed as a part of the new GC process:
 
-- The `end_offset` of the appendable volume should be less than or equal to the `suffix_start_offset` of the upper before appending data to the volume. This should never happen in pracitce, but this validation protects against writing offsets that have already been written to the volume.
+- The appendable volume's `end_offset` should be less than or equal to the upper layer's `suffix_start_offset` before appending data to the volume. This should never happen in pracitce, but this validation protects against writing offsets that have already been written to the volume.
 
-### GC API usage
+### GC API Usage
 
-The new APIs are intended to make using a lower layer with multiple volumes as straightforward as possible. Since every store will start with an empty volume in its lower layer, a client only needs to run a GC as in previous versions of `irmin-pack` by calling `Gc.run`. Data will be moved to the volume in the lower layer automatically. 
+The new APIs are intended to make using a lower layer with multiple volumes as straightforward as possible. Since every store will start with an empty volume in its lower layer, a client only needs to run a GC as in previous versions of `irmin-pack` by calling `Gc.run`. Data will be moved to the lower layer's volume automatically. 
```

Comment:
There might be a missing word here...

"The new APIs are intended to make ______ using a lower layer..."
What does it intend to make?

Reply:
I understand it as direct a translation from french: The APIs intend to make the use of a lower layer that has multiple volumes as easy as possible.

Reply:
Oh now I understand as I reread it again. Thanks.